scripts/emulator-tests/functionsEmulator.spec.ts

```diff
@@ -25,6 +25,7 @@ if ((process.env.DEBUG || "").toLowerCase().indexOf("spec") >= 0) {
 
 const functionsEmulator = new FunctionsEmulator({
   projectId: "fake-project-id",
+  account: undefined,
```

Comment:
Why are we compelled to add this? `undefined` should be the default if it's not set. If we need to specify this, can we add a comment as to why? (same in the file below)

Reply:
I was mostly using this to have `tsc` help me find all the places in the codebase where these args were passed, you're right that I can make it optional now.